### ISSUE TITLE
About set random seed

### ISSUE THREAD

**Issue by kkIIun** [addb-swstarlab/OANet#9]
Hello, 

You only set random seed to 1004 in train_test_split().
Why didn't you set random seed in other function? (e.g. torch, cudnn, numpy etc..)

I think that it results in a different r2 score in every experiment

Thanks for reading!

**Comment by Ychanho** [comment]
Hello,

That's right. To get the same result, fixing the random seed only to train_test_split(), as you commented, may not always be the same result. The code at the time you commented was a version that did not reflect that.

Thank you for the comments!
